File: backend/ws_wa_store_ccadmin/src/main/java/com/ccadmin/app/pucharse/model/dto/PucharseDetLotConfirmDto.java
package com.ccadmin.app.pucharse.model.dto;

import com.ccadmin.app.pucharse.model.entity.PucharseDetEntity;

import java.util.List;

public class PucharseDetLotConfirmDto {

    public PucharseDetEntity pucharseDet;
    public List<PucharseDetEntity> lotDetailList;
    public String WarehouseCod;
}

File: frontend/web_ccadmin_lt/src/app/enterprise/pucharse/model/dto/PucharseDetLotConfirmDto.ts
import { PucharseDetEntity } from "../entity/PucharseDetEntity";

export class PucharseDetLotConfirmDto {

    public pucharseDet: PucharseDetEntity;
    public lotDetailList: PucharseDetEntity[];
    public WarehouseCod: string;

    constructor()
    {
        this.pucharseDet = new PucharseDetEntity();
        this.lotDetailList = [];
        this.WarehouseCod = "";
    }
}
